templates/content/field--oe-document-reference--oe-documents--oe-document-group.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Field template of the document reference document group entity.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% for item in items %}
+  <div{{ not loop.last ? ' class="mb-3"' }}">
```

Comment:
Left-over double quote.
```suggestion
  <div{{ not loop.last ? ' class="mb-3"' }}>
```

Reply:
I remove the left-over double quote.
For the class you mention, I follow what Francesco said about to take the same ewcms has. I think that as long as we don't have the final designs we can leave it like this

Reply:
I would need to look at ewcms, no time for this unless you provide a link :)
But ok to keep until we get updated design.